### PR TITLE
Disabled SSLv3 by default.   Obvious fix.

### PR DESCRIPTION
**Pull request by willhaines**
SSLv3 has been insecure for many years, and should be disabled by default. The proposed settings are from: https://disablessl3.com/postfix.html

### Description
This change improves security of postfix servers using the cookbook.

### Issues Resolved
None

### Check List

- [x] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Review by bmhughes — APPROVED** [review]

**Comment by bmhughes** [comment]
Hi @willhaines,  Can you rebase this on the current master please?

**Comment by bmhughes** [comment]
Also, can you change the CHANGELOG entry to ## Unreleased so the auto-releaser can do it's magic.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of the postfix cookbook.
 
+## 5.3.2 (2020-04-27)
```
Comment (Xorima) [inline CHANGELOG.md:5]:
```suggestion
## Unreleased
```

File: metadata.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 maintainer_email 'cookbooks@chef.io'
 license 'Apache-2.0'
 description 'Installs and configures postfix for client or outbound relayhost, or to do SASL auth'
-version '5.3.1'
+version '5.3.2'
```
Comment (Xorima) [inline metadata.rb:6]:
```suggestion
version '5.3.1'
```

**Review by bmhughes — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [6.0.0](https://github.com/sous-chefs/postfix/releases/tag/6.0.0)

**Pull request merged**